lib/circles/topics/get.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(req, res) {
+    req.pgdb.query("SELECT id, data FROM topics WHERE id = $1", [`${req.circle.circleId}-${req.params.topicName}`], (error, result) => {
```

Comment:
Ok, another note on the id.

From the frontend it has become *hard* to go from a topic id to actually requesting that topic, because `-` is used as delimeter between circleId and topicName, but also insidde circleId.

I *could* fix this, but I think it would be easier to take up my other proposal (change topicName to something that looks better in URL) and then change the delimeter from `-` to `/` between circleId and topicName in the given id.

Additionally, as I mentioned before, the canonical-topic-name (nice url-friendly version) should be returned from the API so it is easier to use.